docs/rst/notes/humble/notes.rst

```diff
@@ -15,45 +15,46 @@ This version ships the following packages:
       - Package version
       - Since Vulcanexus version
     * - eProsima Fast CDR
-      - `v1.0.26 <https://github.com/eProsima/Fast-CDR/releases/tag/v1.0.26>`__
-      - v2.1.0
+      - `v1.0.27 <https://github.com/eProsima/Fast-CDR/releases/tag/v1.0.27>`__
+      - v2.2.0
     * - eProsima Fast DDS
-      - `v2.9.1 <https://fast-dds.docs.eprosima.com/en/latest/notes/notes.html#version-2-9-1>`__
-      - v2.1.0
+      - `v2.10.1 <https://fast-dds.docs.eprosima.com/en/latest/notes/notes.html#version-2-10-1>`__
+      - v2.2.0
     * - eProsima Fast DDS Gen
-      - `v2.3.0 <https://github.com/eProsima/Fast-DDS-Gen/releases/tag/v2.3.0>`__
-      - v2.1.0
+      - `v2.4.0 <https://github.com/eProsima/Fast-DDS-Gen/releases/tag/v2.3.0>`__
+      - v2.2.0
     * - foonathan_memory_vendor
-      - `v1.2.2 <https://github.com/eProsima/foonathan_memory_vendor/releases/tag/v1.2.2>`__
-      - v2.1.0
+      - `v1.3.0 <https://github.com/eProsima/foonathan_memory_vendor/releases/tag/v1.2.2>`__
+      - v2.2.0
     * - micro-ROS Agent
       - `v3.0.4 <https://github.com/micro-ROS/micro-ROS-Agent/blob/humble/micro_ros_agent/CHANGELOG.rst#304-2022-09-28>`__
       - v2.0.4
     * - micro-ROS Setup
       - `v3.1.2 <https://github.com/micro-ROS/micro_ros_setup/blob/humble/CHANGELOG.rst#312-2022-09-28>`__
       - v2.0.4
     * - ROS 2 Monitor
-      - `v1.3.0 <https://fast-dds-monitor.readthedocs.io/en/latest/rst/notes/notes.html#version-v1-3-0>`__
-      - v2.1.0
+      - `v1.4.0 <https://fast-dds-monitor.readthedocs.io/en/latest/rst/notes/notes.html#version-v1-3-0>`__
+      - v2.2.0
     * - ROS 2 Router
-      - `v1.1.0 <https://eprosima-dds-router.readthedocs.io/en/latest/rst/notes/notes.html#version-v1-1-0>`__
-      - v2.1.0
+      - `v1.2.0 <https://eprosima-dds-router.readthedocs.io/en/latest/rst/notes/notes.html#version-v1-1-0>`__
+      - v2.2.0
     * - ROS 2 Shapes Demo
-      - `v2.9.1 <https://eprosima-shapes-demo.readthedocs.io/en/latest/notes/notes.html#version-2-9-1>`__
-      - v2.1.0
+      - `v2.10.1 <https://eprosima-shapes-demo.readthedocs.io/en/latest/notes/notes.html#version-2-9-1>`__
+      - v2.2.0
     * - ROS 2 Statistics Backend
-      - `v0.8.0 <https://fast-dds-statistics-backend.readthedocs.io/en/latest/rst/notes/notes.html#version-0-8-0>`__
-      - v2.0.5
+      - `v0.10.0 <https://fast-dds-statistics-backend.readthedocs.io/en/latest/rst/notes/notes.html#version-0-8-0>`__
+      - v2.2.0
     * - Vulcanexus Fast DDS RMW
       - `vulcanexus-humble <https://github.com/eProsima/rmw_fastrtps/tree/vulcanexus-humble>`__
       - v2.0.3
     * - eProsima Dev Utils
-      - `v0.2.0 <https://github.com/eProsima/dev-utils/releases/tag/v0.2.0>`__
-      - v2.1.0
+      - `v0.3.0 <https://github.com/eProsima/dev-utils/releases/tag/v0.2.0>`__
+      - v2.2.0
 
```

Comment:
Should we include `Fast-DDS-Python` here ?

Reply:
Discusses internally

Reply:
As of now it is not its own package. Let's leave it out while we decide what to do with it.